.gitignore

```diff
@@ -1,3 +1,4 @@
+/.idea
```

Comment:
I don't ignore it globally, I commit .idea (well, most of it) for many work projects and some personal stuff. It's totally worth doing IMO but that's a separate issue.

I certainly don't see the point in *not* including it in project .gitignore files because it makes life easier for contributors... but will remove if people insist.